src/style.css

```diff
@@ -0,0 +1,36 @@
+@import "./styles/title.css";
+@import "./styles/stars.css";
+@import "./styles/price.css";
+@import "./styles/cardWithImage.css";
```

Comment:
Please, use one case of naming files, naming styles etc [link](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841)